hles/extraction/src/main/scala/org/broadinstitute/monster/dap/CslbExtractionPipeline.scala

```diff
@@ -13,8 +13,8 @@ object CslbExtractionPipeline extends ScioApp[Args] {
   // Magic marker for "completed".
   // NB: We are purposefully excluding the recruitment_fields_complete -> 2
   // mapping, as that conflicts with the CSLB data
-  val extractionFilters: Map[String, String] = Map(
-    s"canine_social_and_learned_behavior_complete" -> "2"
+  val extractionFilters: List[FilterDirective] = List(
+    FilterDirective("canine_social_and_learned_behavior_complete", FilterOps.==, "2")
```

Comment:
@quazi-broad should CSLB reflect the DAP pack checks as well?

Reply:
I actually do not think so but we briefly talked about this when we were discussing changing the extraction filters. In our chat you noted "it sure seems like HLES + CSLB extraction criteria are mutually exclusive". I don't think we got around to actually asking Matt that so I'll add that to the list of things to ask him. When we call the CSLB extraction, does it pull all the HLES fields down, or just the CSLB subset. If it's just the CSLB subset, then I do not think we can use the same filters as some of the fields that we check do not exist in the CSLB subset. I'll bring this up with Matt